bucket/xnconvert.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "1.90",
+    "description": "A fast and powerful batch image converter",
+    "homepage": "https://www.xnview.com/en/xnconvert/",
+    "license": "Freeware",
+    "suggest": {
+        "vcredist": "vcredist"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.xnview.com/XnConvert-win-x64.zip",
+            "hash": "c266e34149de6c93a44debda4746622ceaa25b1b65655658a6beaa8c95fb2f7c"
+        },
+        "32bit": {
+            "url": "https://download.xnview.com/XnConvert-win.zip",
+            "hash": "dba3bf1de553e11fac3be90cb4aba210343dc32d85f96956cce89731d970accf"
+        }
+    },
+    "extract_dir": "XnConvert",
+    "pre_install": [
+        "if (Test-Path \"$persist_dir\\xnconvert.ini\") {",
```

Comment:
![image](https://user-images.githubusercontent.com/13260377/116364820-e594c580-a804-11eb-8b7d-7957f64362d4.png)

Reply:
The VC++ download link just needs to be removed, did I understand correctly?